packages/webdriverio/src/commands/element/custom$$.js

```diff
@@ -0,0 +1,46 @@
+/**
+ *
+ * The `customs$` allows you to use a custom strategy declared by using `browser.addLocatorStrategy`
+ *
+ * <example>
+    :pause.js
+    it('should get all the plugin wrapper buttons', () => {
+        browser.url('https://webdriver.io');
+        browser.addLocatorStrategy('myStrat', (selector) => {
+            return document.querySelectorAll(selector)
+        })
+
+        const pluginRowBlock = browser.custom$('myStrat', '.pluginRowBlock')
+        const pluginWrapper = pluginRowBlock.custom$$('myStrat', '.pluginWrapper')
+
+        console.log(pluginWrapper.length) // 4
+    });
+ * </example>
+ *
+ * @alias custom$$
+ * @param {String} strategyName
+ * @param {Any} strategyArguments
+ * @return {Element}
+ */
+import { runFnInFiberContext } from '@wdio/utils/build/shim'
+import { getElements } from '../../utils/getElementObject'
+import { getBrowserObject } from '../../utils'
+
+async function custom$$ (strategyName, strategyArgument) {
```

Comment:
since it is the same implementation you can just require it from the browser directory, no?

Reply:
@christian-bromann It's missing the `parent` element as in the `element.custom$` function. Will fix.